CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
```

Comment:
Probably don't need a CHANGELOG for this... it's an internal change, relating only to testing, so I don't think would even qualify as a patch release.